lux/action/univariate.py

```diff
@@ -104,7 +104,7 @@ def univariate(ldf, *args):
             "long_description": "Temporal displays line charts for all attributes related to datetimes in the dataframe.",
         }
         # Doesn't make sense to generate a line chart if there is less than 3 datapoints (pre-aggregated)
-        if len(ldf) < 3:
+        if ldf.length < 3:
             ignore_rec_flag = True
```

Comment:
I'm a little confused for the change from `len(ldf)` to `ldf.length`. Could you please explain? I don't think Pandas DataFrame has the attribute `length` so I'm worried that this will error, and confused why it won't :P